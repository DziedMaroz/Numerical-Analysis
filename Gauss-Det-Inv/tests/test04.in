6
-1 5 0 0 0 0
0 0 3 -2 0 0
0 0 2 -5 0 0 
-3 6 0 0 0 0
0 0 0 0 4 5
0 0 0 0 3 4 
