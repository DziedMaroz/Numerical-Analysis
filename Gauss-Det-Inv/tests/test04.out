RUNNING ON TEST: tests/test04.in
det (A) =  -99.000
det(A)*A^-1:
   18.00  -45.00   -0.00   -0.00   -0.00   -0.00
   -0.00   -0.00  -33.00   11.00   -0.00   -0.00
   -0.00   -0.00  -66.00   55.00   -0.00   -0.00
   27.00  -18.00   -0.00   -0.00   -0.00   -0.00
    0.00    0.00    0.00    0.00 -396.00  495.00
   -0.00   -0.00   -0.00   -0.00  297.00 -396.00
